src/sass/_permission_confirm_modal.scss

```diff
@@ -0,0 +1,46 @@
+.permission-confirm-modal {
+  .confirm-table {
+    max-height: min(500px, 30vh);
+    overflow-y: auto;
+
+    .remove-identity,
+    .remove-group {
+      text-decoration: line-through;
+    }
+
+    tr {
+      border: none;
+    }
+
+    .first-row {
+      border-top: $border-thin;
+    }
+
+    .display-by-header {
+      position: relative;
+    }
+
+    .display-by-icon {
+      cursor: pointer;
```

Comment:
This should better be a button with bare styling, for a11y and the hover effect. Then we'd get the pointer cursor for free and can remove this line.